src/app/pages/private/LifeListPage/displayLife.tsx

```diff
@@ -1,28 +1,27 @@
+import { useMemo } from 'react';
 import { useListLivesQuery } from '../../../api';
 import NoItemResult from '../../../components/results/NoItemResult';
 
 const DisplayLife = () => {
     const { data, loading } = useListLivesQuery();
+    const lifeList = useMemo(() => (data?.lives || []).map(item => ({ ...item, key: item.id })), [data]);
 
     if (loading) {
         // not yet loaded
         return null;
     }
 
-    const badam = data?.lives || [];
-    console.log(badam)
-    if (badam.length === 0) {
-        return <NoItemResult />
+    if (lifeList.length === 0) {
+        return <NoItemResult />;
     }
 
-    const DisplayData = () => {
-        return (
-            <>
-            {badam.map((life) => <p>{life.id} {life.title} {life.fullName}</p>)}
-            </>
-        )
-    }
-    return <div> <DisplayData /> </div>;
+    const DisplayLives = lifeList.map(life => (
+        <li>
```

Comment:
you `li` element must have a key property, (cf [documentation](https://reactjs.org/docs/lists-and-keys.html))